Core/src/org/sleuthkit/autopsy/corecomponents/DataResultViewerTable.java

```diff
@@ -88,6 +94,7 @@
     // the column started and where it ended up.
     private int startColumnIndex = -1;
     private int endColumnIndex = -1;
+    private OutlineView ov;
```

Comment:
Since we are touching this file, I would prefer to rename "ov" to "outlineView", or even just "view." Highly abbreviated variable names are discouraged by the coding guidelines.